modin/core/execution/ray/common/utils.py

```diff
@@ -138,17 +87,16 @@ def initialize_ray(
                 include_dashboard=False,
                 ignore_reinit_error=True,
                 _redis_password=redis_password,
+                **extra_init_kw,
```

Comment:
Did you checked this case? I can't find that extra parameters could be provided in case of existing cluster https://github.com/ray-project/ray/blob/master/python/ray/_private/worker.py#L1400-L1412

Reply:
> Can you elaborate why it would be useful?

This creates one central place to add more arguments to `ray.init()` instead of copy-pasting them in several different places.

> What the code do you mean?

https://github.com/modin-project/modin/blob/e426ce0331aa4a83f7e0507084c405a8a5256c7d/modin/core/execution/ray/common/utils.py#L170-L177

Reply:
We don't hit this branch in case of cluster init.

Reply:
Huh?.. we either call `ray.init()` or hit this `else:` branch.

We hit `ray.init()` at line 84 (under `if cluster:`) or at line 158 (in `else` branch of that `if cluster`).

Reply:
I was just wondering if we should check the runtime environment after ray.init() in case Modin itself initializes Ray? Because of this.

> Did you check this case? I can't find that extra parameters could be provided in case of existing cluster https://github.com/ray-project/ray/blob/master/python/ray/_private/worker.py#L1400-L1412
>> I hadn't, but apparently extra ones would just get ignored

Reply:
good point!